StortSpelProjekt/Game/src/main.cpp

```diff
@@ -1,5 +1,6 @@
 #include "Engine.h"
 #include "Components/PlayerInputComponent.h"
+#include "EnemyHandler.h"
```

Comment:
Should this be included here? This might be the case, but maybe it should be included somewhere in "game" ?